tests/test_dynamodb.py

```diff
@@ -262,3 +332,17 @@ def test_query_scan_complex(dynamo, complex_query_data):
     res = ComplexKeyModel.query(filter_expr=Rule(f"expires <= '{data_by_expires[2]['expires']}'"))
     res_data = {(m.account, m.sort_date_key): m.dict() for m in res}
     assert res_data == {(d["account"], d["sort_date_key"]): d for d in data_by_expires[:3]}
+
+
+def test_query_with_nested_model(dynamo, nested_query_data):
+    data_by_expires = nested_model_data_generator()
+    res = NestedModel.query(filter_expr=Rule(f"expires <= '{data_by_expires['expires']}'"))
+    res_data = [m.ticket for m in res]
+    assert any(elem is not None for elem in res_data)
```

Comment:
You're checking if it is `not None` here but I think a more specific check is in order. Do we get a dict or an instance of the model?

Reply:
we are getting instance of the model. I want to check  Whether the Ticket Data is serialising and deserialising properly or not using  data and None.So i think checking ticket details like number and creation_time after getting that deserialised is some what meaning full.